src/courtroom.cpp

```diff
@@ -70,6 +70,14 @@ Courtroom::Courtroom(AOApplication *p_ao_app) : QMainWindow()
 
   ui_background = new AOImage(this, ao_app);
 
+  free_brush = *new QBrush(ao_app->get_color("area_free_color", "courtroom_design.ini"));
+  lfp_brush = *new QBrush(ao_app->get_color("area_lfp_color", "courtroom_design.ini"));
+  casing_brush = *new QBrush(ao_app->get_color("area_casing_color", "courtroom_design.ini"));
+  recess_brush = *new QBrush(ao_app->get_color("area_recess_color", "courtroom_design.ini"));
+  rp_brush = *new QBrush(ao_app->get_color("area_rp_color", "courtroom_design.ini"));
+  gaming_brush = *new QBrush(ao_app->get_color("area_gaming_color", "courtroom_design.ini"));
+  locked_brush = *new QBrush(ao_app->get_color("area_locked_color", "courtroom_design.ini"));
```

Comment:
This is a courtroom_design.ini function, therefore it should regenerate in the regen widgets func as opposed to on courtroom creation. This way, the theme makers can properly change colors and press "reload theme" to see their results instead of having to restart the client.

Reply:
fixed